docs/RepoStructure.md

```diff
@@ -0,0 +1,56 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-docker source tree.
+>
+> View the calico-docker documentation for the latest release [here](https://github.com/projectcalico/calico-docker/blob/v0.9.0/README.md).
+<!--- else
+> You are viewing the calico-docker documentation for release **release**.
+<!--- end of master only -->
+
+# Calico Repositories
+
+A number of GitHub repositories provide library functionality and
+orchestration tools that are utilized by Calico.  This repository provides:
+
+- `calico/node`: the Docker image used to run the Calico Felix agent 
+  (responsible for IP routing and ACL programming) and a BGP agent (BIRD) 
+  for distributing routes between hosts.
+- `calicoctl`: the command line tool used for starting a `calico/node` container,
+  configuring Calico policy, adding and removing containers in a Calico network via 
+  orchestration tools, and managing certain network and diagnostic administration.
+
+Additional libraries used by `calicoctl` and orchestration plugin repositories 
+are listed below.
+
+## Calico Libraries
+ - [calico](https://github.com/projectcalico/calico): Implements the Felix 
+   process that interfaces with the Linux kernel to configure routes and ACLs 
+   that control network policy connectivity.  Felix runs as a process within 
+   the `calico/node` container.
```

Comment:
I know that this is a container-specific repo, but is it worth mentioning that this repo also has our OpenStack code?

(Especially since it's somewhat non-intuitive.)